README.rst

```diff
@@ -123,9 +125,9 @@ Involved Information
 
 We appreciate any help! Join us on one of:
 
-- `gitter <https://gitter.im/coala-analyzer/coala/>`
-- `#coala at freenode <webchat.freenode.net/?channels=coala>`
-- `Telegram <https://telegram.me/joinchat/AuL-lwKZ8JLFZiI6SbtQVw>`
+- `gitter <https://gitter.im/coala-analyzer/coala/>`_
+- `#coala at freenode <webchat.freenode.net/?channels=coala>`_
+- `Telegram <https://telegram.me/joinchat/AuL-lwKZ8JLFZiI6SbtQVw>`_
 
 (All channels are linked with gitter. Approach @sils1297 if the link doesn't
 work properly.)
```

Comment:
also, change commit message body tense: ``Adds..``, and end with period please :)